### PR TITLE
Set up Stacked Architecture and linter

### LINKED ISSUE #38: Set up stacked architecture and linter
Set up [Stacked Architecture](https://pub.dev/packages/stacked) and [linter](https://pub.dev/packages/flutter_lints).

### PR DESCRIPTION
**Pull request by nikunjdk**
Set up Stacked Architecture, Moved flashcards related files to temp folder and Added linter for best practices

Closes #38

### CODE REVIEW

File: lib/app/app.dart [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import 'package:freecodecamp/ui/views/browser/browser_view.dart';
+import 'package:freecodecamp/ui/views/home/home_view.dart';
+
+import '../ui/views/startup/startup_view.dart';
+import '../ui/views/website/website_view.dart';
+import 'package:stacked/stacked_annotations.dart';
+import 'package:stacked_services/stacked_services.dart';
+
+// Run 'flutter pub run build_runner build --delete-conflicting-outputs' after any changes in this file to generate updated files
```
Comment (Sembauke) [inline lib/app/app.dart:9]: Hey @nikunjdk,
How exactly did you generate the code? When I use `flutter pub run build_runner build`  nothing seems to happen.

Comment (nikunjdk) [reply lib/app/app.dart:9]: You use it when you need to add new routes or services. It then generates apppp.locator and app.route files

**Comment by Sembauke** [comment]
Did you follow any tutorial to learn this?

**Comment by nikunjdk** [comment]
[This](https://www.filledstacks.com/post/new-setup-for-flutter-stacked-state-management/) is the tutorial for the latest setup. For a deeper understanding you will have to see his previous videos.

File: lib/ui/widgets/dumb/article/article-feed.dart [hunk 2]
```diff
@@ -151,7 +151,7 @@ class _ArticleAppState extends State<ArticleApp> {
     await dotenv.load(fileName: ".env");
 
     String feedUrl =
-        "https://www.freecodecamp.org/news/ghost/api/v3/content/posts/?key=${dotenv.env['NEWSKEY']}&include=tags,authors&page=" +
```
Comment (Sembauke) [inline lib/ui/widgets/dumb/article/article-feed.dart:154]: Can you change this back to its original URL?

**Review by Sembauke — CHANGES_REQUESTED** [review]

**Review by Sembauke — APPROVED** [review]

**Comment by Sembauke** [comment]
Thanks for doing this, we have decided that we are going to merge this first and build on it.

**Pull request merged**